docs/tutorials/services/navigate-with-rover-base.md

```diff
@@ -36,12 +36,20 @@ weight: 5
 # SMEs: Ray Bjorkman, Fahmina
 ---
 
-One key feature of Viam is [the navigation service](/mobility/navigation/), the stateful definition of Viam's [motion service](/mobility/motion/).
 
-Using Navigation, you can queue up user-defined waypoints and your robot will move to them in the order that you specify.
+Using the navigation service, you can queue up user-defined waypoints and your machine will move to them in the order that you specify.
```

Comment:
```suggestion
Using the [navigation service](/mobility/navigation/), you can queue up user-defined waypoints and your machine will move to them in the order that you specify.
```

Reply:
I can also do that - not sure if it duplicates the rest in the intro too much?

Reply:
You mean since it makes the sentence more similar to "add waypoints to specify locations you want your machine to move to, both in the app and programmatically" (line 48)? I think this is still better because the gerund as the first word in the doc just reads oddly to me. Up to you.